File: abc/abc257/a.py
# https://atcoder.jp/contests/abc257/submissions/34773564

n, x = map(int, input().split())
ans = ""
alp = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
cnt = 0
while cnt < 26:
  ans += alp[cnt] * n
  cnt += 1
print(ans[x - 1])

File: abc/abc257/a.java
// https://atcoder.jp/contests/abc257/submissions/43714411

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int x = stdIn.nextInt();

    char[] ptn = new char[26 * n];
    for(int i = 0; i < 26; i++) {
      for(int j = 0; j < n; j++) {
        ptn[i * n + j] = (char)(i + 65);
      }
    }

    System.out.println(ptn[x - 1]);
  }
}
